Dockerfile

```diff
@@ -14,23 +14,16 @@ RUN apt-get update && \
     # Use a fixed apt-get repo to stop intermittent failures due to flaky httpredir connections,
     # as described by Lionel Chan at http://stackoverflow.com/a/37426929/5881346
 RUN sed -i "s/httpredir.debian.org/debian.uchicago.edu/" /etc/apt/sources.list && \
-    apt-get update && apt-get install -y build-essential unzip && \
-    # https://stackoverflow.com/a/46498173
-    conda update -y conda && conda update -y python && \
-    pip install pip==19.0.3 && \
-    apt-get -y install cmake && \
-    /tmp/clean-layer.sh
-
-# Tensorflow doesn't support python 3.7 yet. See https://github.com/tensorflow/tensorflow/issues/20517	
-# Fix to install Tensorflow: Downgrade python 3.7->3.6.6.
-RUN conda install -y python=3.6.6 && \
+    apt-get update && apt-get install -y build-essential unzip cmake && \
+    # Work to upgrade to Python 3.7 can be found on this branch: https://github.com/Kaggle/docker-python/blob/upgrade-py37/Dockerfile
+    conda install -y python=3.6.6 && \
+    pip install --upgrade pip && \
```

Comment:
Using the latest version of pip removed the annoying warning in the logs suggesting that we should upgrade to it.